rmc/utils/mpc.py

```diff
@@ -432,20 +433,57 @@ def _run_glm(
         min_single_aic_var,
     )
 
+    # List possible variable combinations for Joint Regression
+    comb_var = []
+    for k in range(2,6):
+        comb_var += [x for x in combinations(all_var,k)]
+
     logger.info("Running joint (additive interactions only) regression...")
-    add_formula = f"pop_v_path ~ {' + '.join(variables)}"
-    add_X, add_model = _run_glm(add_formula)
+    add_model_res = {}
+    add_model_aic = []
+    add_X = []
+    for var_a in comb_var:
+        logger.info("Running joint (additive interactions) regression for %s...", var_a)
+        formula = f"pop_v_path ~ {' + '.join(var_a)}"
+        X, model = _run_glm(formula)
+        add_X.append(X)
+        add_model_aic.append(model.aic)
+        add_model_res[var_a] = model
+
+    # Find lowest AIC for multiple variable (additive interactions only) regression and
+    # corresponding model
+    min_add_aic = min(add_model_aic)
+    min_add_idx = add_model_aic.index(min_add_aic)
+    min_add_aic_var = comb_var[min_add_idx]
+    min_add_X = add_X[min_add_idx]
+    min_add_formula = f"pop_v_path ~ {' + '.join(min_add_aic_var)}"
 
     logger.info("Running joint regression with all interactions...")
-    mult_formula = f"pop_v_path ~ {' * '.join(variables)}"
-    mult_X, mult_model = _run_glm(mult_formula)
+    mult_model_res = {}
+    mult_model_aic = []
+    mult_X = []
+    for var_m in comb_var:
+        logger.info("Running joint (multiplicative interactions) regression for %s...", var_m)
```

Comment:
Reformatting for line width
```suggestion
        logger.info(
            "Running joint (multiplicative interactions) regression for %s...", var_m
        )
```